cfgov/unprocessed/css/organisms/post-preview.less

```diff
@@ -83,16 +83,45 @@
 */
 
 .o-post-preview {
+    border-bottom: 1px solid @meta-header_border;
+    padding-bottom: unit( 15px / @base-font-size-px, em );
+    display: grid;
+    grid-template-rows: auto;
+    grid-template-columns: auto;
+    grid-template-areas:
+        "header image"
+        "content image";
+
+    .respond-to-max( @bp-xs-max, {
+        grid-template-areas:
+            "header"
+            "image"
+            "content";
+    } );
+    
     + .o-post-preview {
-        margin-top: unit( @grid_gutter-width * 2 / @base-font-size-px, em);
+        margin-top: unit( 30px / @base-font-size-px, em );
```

Comment:
Why this change when we use `@grid_gutter-width` [elsewhere in this PR](https://github.com/cfpb/cfgov-refresh/pull/4811/files#diff-f8dbb35d10b4d3ed1942d92505379790R175)? Are we moving away from that?

Reply:
This could change to `@grid_gutter-width` ( minus the `* 2`), so I can update that.

Reply:
According to @Scotchester we should always use the variable.